components/Layout.jsx

```diff
@@ -1,8 +1,30 @@
-import { Box, Container } from '@mui/material';
+import { Menu as MenuIcon } from '@mui/icons-material';
+import { Box, Button, Container, SwipeableDrawer } from '@mui/material';
```

Comment:
please use single imports 

```suggestion
import Box from '@mui/material/Box';
import Button from '@mui/material/Button';
import Container from '@mui/material/Container';
import SwipeableDrawer from '@mui/material/SwipeableDrawer';
```

this will allow better bundling and also increase the development performance.
See: https://mui.com/material-ui/guides/minimizing-bundle-size/#bundle-size-matters